controls/srg_ctr/SRG-APP-000156-CTR-000380.yml

```diff
@@ -5,8 +5,6 @@ controls:
   title: {{{ full_name }}} must use FIPS-validated SHA-1 or higher hash function
     to provide replay-resistant authentication mechanisms for network access to privileged
     accounts.
-  related_rules:
-  - idp_is_configured
-  - ocp_idp_no_htpasswd
-  - kubeadmin_removed
+  rules:
+  - idp_is_oidc_or_ldap
```

Comment:
Per PR description, shouldn't this SRG reference `ocp_idp_no_htpasswd`?

Adding the rule is redundant, but just want to be sure that its absence is intentional.

Reply:
Actually another really good question.
The answer is no, the rule `ocp_idp_no_htpasswd` satisfies all other standards, but the STIG. For other standards, we really a lot on having a "real" standalone IDP set because then the IDP can enforce things like 2FA or password lockout or account aging. `htpasswd` provides none of that, so for the other standards, all we have to check is that there is an IDP set and that it's not `htpasswd`.

For the STIG however, we standardized on 2 IDPs - LDAP which covers the on-prem case typically connected to MS AD and OIDC. STIG is a bit of a special case in this respect as it serves both as a standard and a reference, so we had to tighten down the selection a bit.

David Anderson will provide you with more details about the history when I'm not available, he had set up the IDP for testing for Mantech/DISA.